### PR TITLE
Add GA to CMS

### PR DESCRIPTION
**Pull request by camfindlay**
Also included a quick improvement to mysql path

### CODE REVIEW

File: mysite/_config.php [hunk 1]
```diff
@@ -12,20 +12,3 @@
...
-
-// Set the current theme. More themes can be downloaded from
-// http://www.silverstripe.org/themes/
-SSViewer::set_theme('simple');
-
-// Disable BasicAuth by default
-BasicAuth::protect_entire_site(false);
-
-// Set the site locale
-i18n::set_locale('en_US');
```
Comment (madmatt) [inline mysite/_config.php]: This hasn't been moved across to `config.yml`

File: mysite/_config.php [hunk 2]
```diff
@@ -12,20 +12,3 @@
...
-
-// Enable nested URLs for this site (e.g. page/sub-page/)
-if (class_exists('SiteTree')) SiteTree::enable_nested_urls();
-
-
-LeftAndMain::require_javascript('mysite/javascript/usabilla.js');
```
Comment (madmatt) [inline mysite/_config.php]: This has disappeared - have you deleted the `usabilla.js` file from the project?

Comment (madmatt) [inline mysite/_config.php]: (The diff is suppressed so I can't see easily :))

File: mysite/javascript/googleanalytics.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+(function(i,s,o,g,r,a,m){ i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-84547-17', 'auto', { 'allowLinker': true });
```
Comment (madmatt) [inline mysite/javascript/googleanalytics.js]: Not really ideal to hard-code this, can it not be added in the CMS? Or is this code actually used to host the demo site (so users would be able to change the GA code if it was in the CMS?)

If so, then it's better to leave it here I guess.

Comment (camfindlay) [inline mysite/javascript/googleanalytics.js]: This is the demo site so we wont want this changed. It was previously added via Page.php requirements so this is a step towards actually treating it like javascript ;)

File: mysite/javascript/usabilla.js [hunk 4]
```diff
@@ -1,3 +0,0 @@
-// include usabilla feedback button
```
Comment (camfindlay) [inline mysite/javascript/usabilla.js]: this was deleted.

File: mysite/_config.php (hunk 2 above)
Comment (camfindlay) [inline mysite/_config.php]: Deleted the usabilla.js file and moved from _config.php to yaml config to include new googleanalytics.js

File: mysite/_config.php (hunk 1 above)
Comment (camfindlay) [inline mysite/_config.php]: This is default in 3.3.x anyway

**Comment by madmatt** [comment]
Looks good to me with your changes, but I don't have merge rights on this repo :)

**Pull request merged**